src/Console/Application.php

```diff
@@ -34,13 +40,60 @@ class Application extends BaseApplication
      */
     private $container;
 
+    /**
+     * @var SymfonyStyle
+     */
+    private $io;
+
+    /**
+     * @var bool
+     */
+    private $isXdebugLoaded;
+
+    /**
+     * @var bool
+     */
+    private $isDebuggerDisabled;
+
     public function __construct(Container $container, string $name = self::NAME, string $version = self::VERSION)
     {
         $this->container = $container;
+        $this->isDebuggerDisabled = empty((string) getenv(XdebugHandler::ENV_DISABLE_XDEBUG));
 
         parent::__construct($name, $version);
     }
 
+    public function run(InputInterface $input = null, OutputInterface $output = null)
+    {
+        if (null === $input) {
+            $input = new ArgvInput();
+        }
+
+        if (null === $output) {
+            $output = new ConsoleOutput();
+        }
+
+        $this->io = new SymfonyStyle($input, $output);
+
+        if (PHP_SAPI === 'phpdbg') {
+            $this->io->note(sprintf(self::RUNNING_WITH_NOTE, PHP_SAPI));
+        } elseif ($this->isXdebugLoaded) {
+            $this->io->note(sprintf(self::RUNNING_WITH_NOTE, 'xdebug'));
+        }
+
+        $xdebug = new XdebugHandler(new ConfigBuilder(sys_get_temp_dir()));
+        $xdebug->check();
+        unset($xdebug);
```

Comment:
why do you need to unset it?

Reply:
I hope it'll free memory. to be honest I've just moved it from the bootstrap.php as is

Reply:
since this is a local var in the class method, I suppose it would be garbage collected automatically